titus-supplementary-component/job-activity-history/src/main/resources/db/migration/V1__initial_setup.sql

```diff
@@ -16,22 +16,30 @@
 
 CREATE SCHEMA IF NOT EXISTS jobactivity;
 
-
 CREATE TABLE IF NOT EXISTS jobactivity.jobs
 (
-  job_id          VARCHAR (64) NOT NULL,
-  create_time     TIMESTAMP NOT NULL,
-  record_time     TIMESTAMP NOT NULL,
-  job_record_json jsonb,
-  PRIMARY KEY (job_id)
+    job_id          VARCHAR(64) NOT NULL,
+    record_time     TIMESTAMP   NOT NULL,
+    job_record_json varchar(64),
```

Comment:
Ideally this should be unlimited size or very high (128KB). We might also consider gziping it to save space.